### PR TITLE
PXN-3510 Tracks Alignment Result Abort

### PR DESCRIPTION
**Pull request by Esteban-ML**
## What have changed?
Add missing Abort tracks

- /px_checkout/result/success/abort
- /px_checkout/result/further_action_needed/abort
- /px_checkout/result/error/abort

- /payment_congrats/result/success/abort
- /payment_congrats/result/further_action_needed/abort
- /payment_congrats/result/error/abort

## How to test:
Tap the Close Button located at the top left corner of the congrats screens.

_Doesn’t work for the Example Congrats_

## Prints for layout changes:
No UI Changes

### CODE REVIEW

File: MercadoPagoSDK/MercadoPagoSDK/Models/BusinessResult/PXBusinessResultViewModel+Tracking.swift [hunk 1]
```diff
@@ -100,6 +100,23 @@ extension PXBusinessResultViewModel: PXViewModelTrackingDataProtocol {
         return debinProperties
     }
 
+    func getCloseButtonTrack() -> PXResultTrackingEvents? {
```
Comment (jlainog-meli) [inline MercadoPagoSDK/MercadoPagoSDK/Models/BusinessResult/PXBusinessResultViewModel+Tracking.swift:103]: Why is this optional? I think it is not needed to be optional.

File: MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXPaymentCongratsViewModel.swift [hunk 2]
```diff
@@ -85,6 +85,24 @@ class PXPaymentCongratsViewModel {
 }
 
 extension PXPaymentCongratsViewModel: PXNewResultViewModelInterface {
+    func getCloseButtonTrack() -> PXResultTrackingEvents? {
```
Comment (jlainog-meli) [inline MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXPaymentCongratsViewModel.swift:88]: looks line this logic is duplicated, can we extract this logic to one place instead ?

File: MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXResultViewModel.swift [hunk 3]
```diff
@@ -305,6 +309,23 @@ extension PXResultViewModel {
 }
 
 extension PXResultViewModel: PXViewModelTrackingDataProtocol {
+    func getCloseButtonTrack() -> PXResultTrackingEvents? {
```
Comment (jlainog-meli) [inline MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXResultViewModel.swift:312]: same here

**Review by jlainog-meli — CHANGES_REQUESTED** [review]
LGTM but you have duplicated code. try extract it and reused it.

File: MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXPaymentCongratsViewModel.swift (hunk 2 above)
Comment (Esteban-ML) [reply MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXPaymentCongratsViewModel.swift]: Done!

File: MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXResultViewModel.swift (hunk 3 above)
Comment (Esteban-ML) [reply MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXResultViewModel.swift]: Done

**Review by jlainog-meli — APPROVED** [review]

**Review by rgrajalesmeli — APPROVED** [review]
LG!

**Pull request merged**